README.md

```diff
@@ -143,6 +159,22 @@ apt install podman rootlesskit slirp4netns
 
 should be enough to get you a working Podman setup.
 
+You can then either:
+
+- Use `build-podman.sh` as described above, which clones and builds
+  the tkey-libs libraries as well.
+
+- Download [pre-built versions of the tkey-libs
+  libraries](https://github.com/tillitis/tkey-libs/releases) and
+  define `MOUNTLIBDIR` to where you unpacked the tkey-libs, something
+  like:
+
+  ```
+  make LIBDIR=/tkey-libs MOUNTLIBDIR=$HOME/Downloads/tkey-libs-v0.1.1 podman
```

Comment:
I'm not sure if I'm missing something (or if it is differences between darwin and linux), but I get no difference in my output if i'm using 

```
make LIBDIR=/tkey-libs MOUNTLIBDIR=$HOME/Downloads/tkey-libs-v0.1.1 podman
```
or
```
make LIBDIR=$HOME/Downloads/tkey-libs-v0.1.1 podman
```

So at least for me, the `MOUNTLIBDIR` seems unnecessary?

Reply:
Ha! So it is. `LIBDIR` inside the container is set to `LIBDIR ?= $(P)/../tkey-libs` which just happens to be `/tkey-libs` where I mounted the external tkey-libs directory.

I'll remove the superfluous variable.